### ISSUE TITLE
Support scrolling the filetree panel based targets

### ISSUE THREAD

**Issue by fidgetingbits** [paul-schaaf/talon-filetree#25]
It would be nice to have something similar to cursorless/rango where you can use crown, center, bottom to automatically scroll the file tree panel. I've found myself wanting this on huge code bases.

**Comment by david-tejada** [comment]
Yes, that would be nice. Unfortunately, there is no API for scrolling a tree view. The only thing you can do is to select a file/folder at the top or bottom using `tree select`, this will cause the selected element to scroll to the center.

**Comment by fidgetingbits** [comment]
That's a nice suggestion, thanks. Is this generally the type of thing where it's worth pushing for that functionality/API to get added to vscode? I'm not too familiar with how hard it is to get things added.

**Comment by david-tejada** [comment]
We can give it a try. I don't think we'll have much success, though, as niche as this is. I'll see if I find some time to write an issue.
